src/terms/nonlocal.jl

```diff
@@ -178,22 +178,34 @@ end
 """
 Build form factors (Fourier transforms of projectors) for an atom centered at 0.
 """
-function build_form_factors(psp, qs)
-    qnorms = norm.(qs)
-    T = real(eltype(qnorms))
-    # Compute position-independent form factors
-    form_factors = zeros(Complex{T}, length(qs), count_n_proj(psp))
-    count = 1
-    for l in 0:psp.lmax, m in -l:l
-        prefac_lm = im^l .* ylm_real.(l, m, qs)
-        n_proj_l = size(psp.h[l + 1], 1)
+function build_form_factors(psp, G_plus_ks)
```

Comment:
should probably be qs instead of G_plus_ks

Reply:
our convention is to use q for a generic fourier-space vector, so that q=k+G recurs quite a bit

Reply:
This is btw a very unfortunate convention, as q is also the phonon wavevector, which is in the discrete Brillouin zone... I'm not sure if we should change this and if yes to what; cc @epolack @mfherbst

Reply:
Ok. Then let's stick to `q` for now.